source/AMDSClient.h

```diff
@@ -56,6 +56,7 @@ private slots:
 	QSpinBox* amptekIndex;
 	QComboBox *bufferNameComboBox_;
 	QCheckBox* includeStatusDataCheckbox;
```

Comment:
This file is replaced with AMClientUi.h in a later PR and the comments will be applied there.